src/bindings/python/__init__.py

```diff
@@ -0,0 +1,4 @@
+try:
+    from .python_valhalla import *
+except ModuleNotFoundError:
+    from python_valhalla import *
```

Comment:
is the module actually named `python_valhalla` or can a general user of this module `import valhalla` instead? i much prefer the latter. at least that is what it was called when we had the boost python bindings (iirc).

Reply:
It's a little convoluted tbh, but it's `import valhalla` still, since the module gets installed to `/usr/.../site-packages/valhalla`. The init.py just imports everything from the cpp lib `python_valhalla` (which resides in the same folder) and is just the target name in CMake. It was actually the same with boost.python I think.

Reply:
But before the `python_valhalla` cpp lib was installed directly to `site-packages` instead of `site-packages/valhalla`, so I threw in the try/except just in case.